prow/config.yaml

```diff
@@ -12,6 +12,23 @@ presets:
       - name: sa-vm-kyma-integration
         mountPath: /etc/credentials/sa-vm-kyma-integration
         readOnly: true
+  - labels:
+      preset-sa-gke-kyma-integration: "true"
+    env:
+      - name: GCLOUD_SERVICE_KEY_PATH
+        value: /etc/credentials/sa-gke-kyma-integration/service-account.json
+      - name: GCLOUD_PROJECT_NAME
```

Comment:
there will be preset introduced by Suleyman's PR: https://github.com/kyma-project/test-infra/pull/98/files#diff-4